scripts/emoji/translate.ts

```
@@ -0,0 +1,336 @@
+import { promises as fs } from "fs";
+import axios, { AxiosResponse } from "axios";
+import dotenv from "dotenv";
+import { fileURLToPath } from "url";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)